pyQuARC/code/custom_validator.py

```diff
@@ -102,3 +90,64 @@ def bounding_coordinate_logic_check(coordinates_dictionary):
         )
 
         return result
+
+    @staticmethod
+    def presence_check(*field_values):
+        validity = False
+        value = None
+
+        for field_value in field_values:
+            if value := field_value and field_value.strip():
```

Comment:
This seems a bit strange.

`if field_value and field_value.strip()`

A comment could help.

Reply:
basically checking if it is not just a whitespace